scripts/edit-datapackage-json.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+
+import seedcase_sprout.core as sp
+
+properties = sp.PackageProperties(
+    description=(
+        "Dung beetles relocate vertebrate feces under the soil surface, "
+        "and this behavior has many ecological consequences. In tropical forests, "
+        "those seeds that are defecated by mammals and subsequently buried "
+        "by dung beetles are less likely to suffer predation."
+    ),
+)
+
+package_path = Path(__file__).resolve().parent.parent / "datapackage.json"
```

Comment:
Once my PR has been merged, these can be updated to use the `path_` functions. For now, just making a comment here!